server/src/main/java/org/elasticsearch/action/support/master/TransportMasterNodeAction.java

```diff
@@ -98,7 +98,9 @@ protected boolean localExecute(Request request) {
 
     @Override
     protected void doExecute(final Request request, ActionListener<Response> listener) {
-        new AsyncSingleAction(request, listener).start();
+        ClusterState state = clusterService.state();
+        logger.trace("starting processing request [{}] with cluster state version [{}]", request, state.version());
```

Comment:
we removed Task in https://github.com/crate/crate/commit/d872104f5b124a54c2725044c45696ea7196446d#diff-e7c445981d60f65376f04699ec430437d9b1c78fac6b5ef582f39f84e43de9ceL117
so no if `(task != null)` check here